.github/workflows/ci_s2ntests.yml

```diff
@@ -0,0 +1,377 @@
+        with:
+          project-name: 's2nGithubCodebuild'
+          env-vars-for-codebuild: |
+            BUILD_S2N,
+            CODECOV_IO_UPLOAD,
+            GCC_VERSION,
+            S2N_COVERAGE,
+            S2N_LIBCRYPTO,
+            TESTS,
+            requester,
+            event-name
+        env:
+          BUILD_S2N: true
+          CODECOV_IO_UPLOAD: true
+          S2N_LIBCRYPTO: ${{ matrix.openssl_version }}
+          S2N_COVERAGE: true
+          TESTS: "unit"
+          GCC_VERSION: ${{ matrix.gcc_version }}
+          requester: ${{ github.actor }}
+          event-name: ${{ github.event_name }}
+  valgrind:
+    needs: [unit, integ, asan, sawhmac, sawother]
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        openssl_version:
+          - openssl-1.0.2
+          - openssl-1.0.2-fips
+          - openssl-1.1.1
+        gcc_version:
+          - 6
+          - 9
+      fail-fast: true
+    steps:
+      - uses: actions/setup-node@v1
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: S2n Valgrind CodeBuild
+        uses: aws-actions/aws-codebuild-run-build@master
+        with:
+          project-name: 's2nGithubCodebuild'
+          env-vars-for-codebuild: |
+            BUILD_S2N,
+            GCC_VERSION,
+            S2N_LIBCRYPTO,
+            TESTS,
+            requester,
+            event-name
+        env:
+          BUILD_S2N: true
+          S2N_LIBCRYPTO: ${{ matrix.openssl_version }}
+          TESTS: "valgrind"
+          GCC_VERSION: ${{ matrix.gcc_version }}
+          requester: ${{ github.actor }}
+          event-name: ${{ github.event_name }}
+  fuzzers:
+    needs: [valgrind, unit, integ, asan, sawhmac, sawother]
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        openssl_version:
+          - openssl-1.0.2-fips
+          - openssl-1.1.1
+      fail-fast: true
+    steps:
+      - uses: actions/setup-node@v1
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: S2n Fuzz CodeBuild
+        uses: aws-actions/aws-codebuild-run-build@master
+        with:
+          project-name: 's2nGithubCodebuild'
+          env-vars-for-codebuild: |
+            FUZZ_TIMEOUT_SEC,
+            LATEST_CLANG,
+            S2N_LIBCRYPTO,
+            TESTS,
+            requester,
+            event-name
+        env:
+          LATEST_CLANG: true
+          S2N_LIBCRYPTO: ${{ matrix.openssl_version }}
+          S2N_COVERAGE: true
+          TESTS: "fuzz"
+          FUZZ_TIMEOUT_SEC: 300
```

Comment:
Did we have a 14 minute timeout before? Does this give enough time for tests to complete?

Reply:
It was 120 seconds previously, so a slight bump.